caffe2/ideep/operators/conv_pool_base_op.h

```diff
@@ -43,7 +40,7 @@ class IDEEPConvPoolOpBase : public ConvPoolOpBase<IDEEPContext> {
     InferOutputSize(
         input_Tdims,
         output_channel,
-        order_,
+        StorageOrder::NCHW, //order_,
```

Comment:
In MKL-DNN, the shape is always unchanged, no matter what kind of memory layout the tensor is taken.
In other words, the order of shape is always same as NCHW.
Therefore, the order_ of cafffe2 operator should not change the result of InferOutputSize.